front/app/community/[siren]/comparison/[comparedSiren]/ComparingFiche/ComparingFiche.tsx

```diff
@@ -0,0 +1,22 @@
+import { Community } from '@/app/models/community';
+import { TransparencyScoreBar } from '@/components/TransparencyScore/TransparencyScore';
+import { TransparencyScore } from '@/components/TransparencyScore/constants';
+
+import { CommunityDetails } from '../../../components/CommunityDetails';
+
+type ComparingFicheProps = {
+  community: Community;
+};
+
+export function ComparingFiche({ community }: ComparingFicheProps) {
+  // TODO - connect to community when in db
+  const scoreTODO = TransparencyScore.A;
+
+  return (
+    <div className='flex flex-col rounded bg-gray-100 p-6' style={{ rowGap: 50 }}>
```

Comment:
Utiliser plutôt space-y à la place de rowGap pour rester dans du tailwind
```suggestion
    <div className='flex flex-col rounded bg-gray-100 p-6 space-y-[50px]'>
```